runtime/src/main/resources/packaged/configuration.yml

```diff
@@ -1,7 +1,10 @@
+flux.Shard.Config:
+  master.slave.pair.list: { "masterSlavePairList" : [ { "master":{ "shardId":1, "ip": "127.0.0.1", "shardKeys": ["0","1","2","3","4","5","6","7","8","9","a","b","c","d","e","f"] },"slave":{ "shardId":2, "ip":"127.0.0.1", "shardKeys": ["0","1","2","3","4","5","6","7","8","9","a","b","c","d","e","f"] }}  ]  }
```

Comment:
Also we should be validating the cases like range overlaps, startOfRang < EndOfRange etc. and error out if there is something wrong